torchao/quantization/GPTQ.py

```diff
@@ -1022,13 +1024,15 @@ def __init__(
         precision: torch.dtype = torch.float32,
         scales_precision: torch.dtype = torch.float32,
         device: torch.device = torch.device("cpu"),
+        mapping_type: MappingType = MappingType.SYMMETRIC_MAX_POS_NEG
```

Comment:
actually why is this set to the new type? I think it will be safer to keep the old one?